stdlib/public/stubs/Stubs.cpp

```diff
@@ -249,13 +255,60 @@ extern "C" uint64_t swift_float80ToString(char *Buffer, size_t BufferLength,
 }
 
 /// \param[out] LinePtr Replaced with the pointer to the malloc()-allocated
-/// line.  Can be NULL if no characters were read.
+/// line.  Can be NULL if no characters were read. This buffer should be
+/// freed by the caller if this function returns a positive value.
 ///
 /// \returns Size of character data returned in \c LinePtr, or -1
 /// if an error occurred, or EOF was reached.
 ssize_t swift::swift_stdlib_readLine_stdin(char **LinePtr) {
+#if defined(_MSC_VER)
+  if (LinePtr == nullptr)
+    return -1;
+
+  int Capacity = 0;
```

Comment:
Probably does not matter in practice, but `Capacity` should probably be declared as `ssize_t`.